packages/consts/src/consts.ts

```diff
@@ -114,6 +119,19 @@ export const USERNAME = {
     REGEX: /^[a-zA-Z0-9_.-]{3,30}$/,
 };
 
+export const EMAIL = {
+    MAX_LENGTH: 254, // see https://www.rfc-editor.org/errata_search.php?rfc=3696&eid=1690
+    REGEX: EMAIL_REGEX,
+};
+
+/**
+ * Profile name (such as organization or first / last name) constraints.
+ */
+export const PROFILE_NAME = {
+    MAX_LENGTH: 50,
+    REGEX: /^(?!.*:\/\/)[^@><]*$/, // Prohibits usage of @, <, > and ://
+};
```

Comment:
Question: Do we want to have this in apify-shared, even if it's just used in apify-core?